components/brave_wallet/browser/keyring_service.cc

```diff
@@ -2018,151 +1968,182 @@ void KeyringService::MaybeMigratePBKDF2Iterations(const std::string& password) {
   }
 }
 
-void KeyringService::StopAutoLockTimer() {
-  auto_lock_timer_->Stop();
-}
-
-void KeyringService::ResetAutoLockTimer() {
-  if (auto_lock_timer_->IsRunning()) {
-    auto_lock_timer_->Reset();
-  } else {
-    size_t auto_lock_minutes =
-        (size_t)profile_prefs_->GetInteger(kBraveWalletAutoLockMinutes);
-    auto_lock_timer_->Start(FROM_HERE, base::Minutes(auto_lock_minutes), this,
-                            &KeyringService::OnAutoLockFired);
+void KeyringService::MaybeMigrateToWalletMnemonic(const std::string& password) {
```

Comment:
We had originally talked about a way to get rid of some of these migrations, but @supermassive thought this approach would be a bit easier to deal with. this approach LGTM